docs/sources/configuration/traces-config.md

```diff
@@ -258,7 +258,13 @@ tail_sampling:
   # Time that to wait before making a decision for a trace.
   # Longer wait times reduce the probability of sampling an incomplete trace at
   # the cost of higher memory usage.
-  decision_wait: [ <duration> | default="5s" ]
+  [ decision_wait: <duration> | default="5s" ]
+
+  # Optional, number of traces kept in memory
+  [ num_traces: <int> | default="50000" ]
+
+  # Optional, expected number of new traces (helps in allocating data structures)
+  [ expected_new_traces_per_sec: <int> | default="0" ]
```

Comment:
For consistency there should be spacing surrounding the equal signs and default values should match what a user would enter rather than always being placed in quotes:

```suggestion
  [ decision_wait: <duration> | default = "5s" ]

  # Optional, number of traces kept in memory
  [ num_traces: <int> | default = 50000 ]

  # Optional, expected number of new traces (helps in allocating data structures)
  [ expected_new_traces_per_sec: <int> | default = 0 ]
```